### PR TITLE
714 Add LabelToMask transform

### LINKED ISSUE #714: Add LabelToMask transform
**Is your feature request related to a problem? Please describe.**
Need a transform to convert different kinds of segmentation labels to binary masks to pre-process image for other tasks.
labels can be 1 channel or multi-channels in One-Hot format.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #714 .

### Description
This PR implemented the `LabelToMask` and `LabelToMaskd` transforms.
They can be used to convert the labels to mask and be helpful for other tasks.

### Status
**Ready**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### CODE REVIEW

File: monai/transforms/utility/array.py [hunk 1]
```diff
@@ -372,3 +373,39 @@ def __call__(self, img, func: Optional[Callable] = None):
             return self.func(img)
         else:
             raise RuntimeError("neither func or self.func is callable.")
+
+
+class LabelToMask(Transform):
+    """
+    Convert label to mask for other tasks.
```
Comment (wyli) [inline monai/transforms/utility/array.py:380]: could you update the docstring, it's unclear when and how to use this transform. also perhaps `compose_channels` could be `merge_channels`

Comment (Nic-Ma) [reply monai/transforms/utility/array.py]: updated in the latest commit.
Thanks.

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Thanks for your review.
I updated the code and doc-string according to your comments.
Could you please help review it again?
Thanks.

**Pull request merged**